### PR TITLE
WIP v2: Documentation + human output arrangments

### PR DESCRIPTION
**Pull request by nvuillam**
Update README documentation

### CODE REVIEW

**Comment by nvuillam** [comment]
cc @RubenHalman :)

**Comment by nvuillam** [comment]
@RubenHalman any chance to merge this PR without releasing a new version ? :)

**Comment by RubenHalman** [comment]
Hi Nicolas. If this is only Docs, we can merge without publishing. However you also changed pieces of code that would need to be published before taking effect.

File: src/commands/flow/scan.ts [hunk 1]
```diff
@@ -83,27 +97,36 @@ export default class scan extends SfdxCommand {
...
     const parsedFlows: Flow[] = await ParseFlows(flowFiles);
+    this.ux.startSpinner(`Parsed ${flowFiles.length} flows, processing analysis`);
     let scanResults: ScanResult[];
     if (this.userConfig && Object.keys(this.userConfig).length > 0) {
       scanResults = core.scan(parsedFlows, this.userConfig);
     } else {
       scanResults = core.scan(parsedFlows);
     }
 
+console.log(JSON.stringify(scanResults));
```
Comment (RubenHalman) [inline src/commands/flow/scan.ts:112]: Rm console log please

**Review by RubenHalman — CHANGES_REQUESTED** [review]
Lgtm and if you've tested it we can merge and publish, only let's keep the deployment stuff out of this pr please.

File: src/commands/flow/scan.ts (hunk 1 above)
Comment (nvuillam) [reply src/commands/flow/scan.ts:112]: Work in progress, I wouldn't dare to let it here haha ^^

**Pull request merged**